README.md

```diff
@@ -31,43 +31,81 @@ After installing you can start karmabot by using the command
 karmabot
 ```
 
-However, you need to supply some settings prior to this.
+However, you need to some setup and supply some settings prior to this.
 
-### Settings
+### Setup
+
+For app creation and tokens please follow the [slack-bolt guide](https://slack.dev/bolt-python/tutorial/getting-started) and enable [socket mode](https://slack.dev/bolt-python/concepts#socket-mode).
+
+#### Settings
 
 By default we will look for a `.karmabot` file in the directory you used the `karmabot` command. The file should supply the following information.
 
 ```env
+# Slack bot app
+KARMABOT_SLACK_BOT_TOKEN=
+KARMABOT_SLACK_APP_TOKEN=
+
+# Workspace
 KARMABOT_SLACK_USER=
-KARMABOT_SLACK_TOKEN=
-KARMABOT_SLACK_INVITE_USER_TOKEN=
-KARMABOT_DATABASE_URL=
 KARMABOT_GENERAL_CHANNEL=
 KARMABOT_ADMINS=
+
+# Backend
+KARMABOT_DATABASE_URL=
```

Comment:
Can we change this to `DATABASE_URL` which Heroku uses by default when provisioning a Postgres DB?

Reply:
I would rather not do that as karmabot is not a Heroku specific application.  

We use env variables or the settings file. If the requirement for the name would be "DATABASE_URL", this could potentially clash with other system variables, e.g., some other tool that decrales a DATABASE_URL env variable. Prefixing everything with KARMABOT_ makes this operation save.

Reply:
OK I do like the prefixing, thanks